package.json

```diff
@@ -60,6 +60,7 @@
     "leven": "^2.1.0",
     "lodash": "^4.17.15",
     "lru-cache": "^4.1.1",
+    "moment": "^2.24.0",
```

Comment:
I was not aware of the other date library is being used. I use the moment libaray just to convert months in different formats to the corresponding number. For example both of ``moment().month("Nov").format("M")`` and ``moment().month("November").format("M")`` will yield to "11".

I could not find a way of doing the same in ``date-fns``. Probably putting a second libarary responsible of dates would not sound good. In that case I can implement that conversion by myself. In that case I need to know the possible moth formats. So far I have seen only things like "Nov" or "11". Do you know if having the full month name like "November" or any other format is possible (if there is any other format I am not sure even how I was suing ``moment`` library would be covering that though)?

Reply:
Would you please post an example date string from the data?

Reply:
https://dtd.nlm.nih.gov/ncbi/pubmed/doc/out/180101/el-PubDate.html 

> Element: Month
> Contains the value of the month in a date. It will be recorded as either the numeric value or the first three letters of the month's name.

Reply:
> Element: PubDate
> ( ( Year, ( ( Month, Day? ) | Season )? ) | MedlineDate )

Reply:
If you have individual variables for things like `month` in `MM` format, you can just use `parse()` with the appropriate format.

https://date-fns.org/v2.12.0/docs/parse

The format strings are similar to what you may have already seen with the Unix `date` CLI: http://man7.org/linux/man-pages/man1/date.1.html

Reply:
Okay it says that either numeric or the first three letters. Then I can make it work as the following: ``parse('Nov', 'MMM', new Date()).getMonth() + 1`` which yields to 11 as expected.